client/src/components/Sidecart/Sidecart.jsx

```diff
@@ -0,0 +1,60 @@
+import "./Sidecart.css";
+import { Button, Drawer, Typography } from "@mui/material";
+import { useShoppingCart } from "../../context/ShoppingCartContext";
+import Stack from "@mui/material/Stack";
+import Box from "@mui/material/Box";
+import { Link as RouterLink } from "react-router-dom";
+
+import CartProductList from "../CartProductList/CartProductList";
+
+export function SideCart({ isCartOpen }) {
+  const { closeCart, getCartTotalPrice } = useShoppingCart();
+
+  return (
+    <Drawer
+      anchor="right"
+      open={isCartOpen}
+      onClose={closeCart}
+      PaperProps={{
+        sx: {
+          width: 500,
+        },
+      }}
+    >
+      <Box sx={{ padding: "15px" }}>
+        <Stack spacing={3} direction={"column"}>
+          <Stack spacing={2}>
+            <CartProductList></CartProductList>
```

Comment:
Can write just  <' CartProductList  / '> instead, no?
Without the '  ' - git does not let me write it

Reply:
דחפתי תיקון, אל תכניס עדיין לmain